2697. Lexicographically Smallest Palindrome/solution.py

```diff
@@ -1,6 +1,20 @@
 class Solution:
     # takes in the string s and turns it into a palindrome
     # by replacing as few letters as possible. breaks tie by
-    # checking the lexicographical value of the current letter
+    # checking the lexicographical value of the current letter.
```

Comment:
You still don't explain what "lexographical" means and whether it's the lexographically smallest or biggest string.

Reply:
Cody, your tone is negative. Please phrase your comment as a suggestion. You can paste what you wrote to explain lexographical as a suggestion to Winry.

Reply:
Hmm... You guys both said "Lexographical" instead of "Lex**ic**ographical".